scripts/anvil_tdr_ingest/findability_subset/findability_subset_schema.json

```diff
@@ -0,0 +1,278 @@
+        {"name": "data_use_permission", "datatype": "string", "array_of": true},
+        {"name": "owner", "datatype": "string", "array_of": true},
+        {"name": "principal_investigator", "datatype": "string", "array_of": true},
+        {"name": "registered_identifier", "datatype": "string", "array_of": true},
+        {"name": "title", "datatype": "string", "array_of": false},
+        {"name": "data_modality", "datatype": "string", "array_of": true}
+      ],
+      "primaryKey": ["dataset_id"],
+      "partitionMode": "none",
+      "datePartitionOptions": null,
+      "intPartitionOptions": null
+    },
+    {
+      "name": "diagnosis",
+      "columns": [
+        {"name": "diagnosis_id", "datatype": "string", "array_of": false, "required": true},
+        {"name": "disease", "datatype": "string", "array_of": true},
+        {"name": "diagnosis_age_unit", "datatype": "string", "array_of": false},
+        {"name": "diagnosis_age_lower_bound", "datatype": "float", "array_of": false},
+        {"name": "diagnosis_age_upper_bound", "datatype": "float", "array_of": false},
+        {"name": "onset_age_unit", "datatype": "string", "array_of": false},
+        {"name": "onset_age_lower_bound", "datatype": "float", "array_of": false},
+        {"name": "onset_age_upper_bound", "datatype": "float", "array_of": false},
+        {"name": "phenotype", "datatype": "string", "array_of": true},
+        {"name": "phenopacket", "datatype": "string", "array_of": true}
+      ],
+      "primaryKey": ["diagnosis_id"],
+      "partitionMode": "none",
+      "datePartitionOptions": null,
+      "intPartitionOptions": null
+    },
+    {
+      "name": "donor",
+      "columns": [
+        {"name": "donor_id", "datatype": "string", "array_of": false, "required": true},
+        {"name": "organism_type", "datatype": "string", "array_of": false},
+        {"name": "part_of_dataset_id", "datatype": "string", "array_of": false},
+        {"name": "phenotypic_sex", "datatype": "string", "array_of": false},
+        {"name": "reported_ethnicity", "datatype": "string", "array_of": true},
+        {"name": "genetic_ancestry", "datatype": "string", "array_of": true},
+        {"name": "diagnosis_id", "datatype": "string", "array_of": true}
+      ],
+      "primaryKey": ["donor_id"],
+      "partitionMode": "none",
+      "datePartitionOptions": null,
+      "intPartitionOptions": null
+    },
+    {
+      "name": "file",
+      "columns": [
+        {"name": "file_id", "datatype": "string", "array_of": false, "required": true},
+        {"name": "data_modality", "datatype": "string", "array_of": true},
+        {"name": "file_format", "datatype": "string", "array_of": false},
+        {"name": "file_ref", "datatype": "fileref", "array_of": false},
+        {"name": "reference_assembly", "datatype": "string", "array_of": true},
+        {"name": "label", "datatype": "string", "array_of": false}
```

Comment:
I think there should also be attributes for file size and file checksums.
FYI, the BioData Catalyst minimal metadata model for files is available here for reference/comparison:
https://docs.google.com/document/d/1U3uh4P8oHKDvKTwi5HSwmije0q0MuxiSp-i3-6cSqIw/edit

Reply:
The "findability" naming convention we are using is perhaps a bit misleading. Essentially, this schema should encompass anything we want to capture consistently across contributing datasets. I agree that these two items are potentially valuable to capture and have added them to https://broadworkbench.atlassian.net/browse/DI-121 (where I'm consolidating potential updates to the FSS schema).